programs/flatten.js

```diff
@@ -0,0 +1,15 @@
+// Problem 3: Complete the flatten function that takes a JS Object, returns a JS Object in flatten format (compressed)
+function flatten(unflatObject) {
+  var output = {};
+  for (var iteration in unflatObject) {
```

Comment:
I was a 100% sure this would be copy pasted here :(
Please correct based on the comment on assignment 1 PR

Reply:
changes iteration with property. commit ID - 5fa48e1abb44c2c786a9b3f623d54d45859ed4ab